OpenSubtitlesHandler/Models/ApiResponse.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+
+namespace OpenSubtitlesHandler.Models
+{
+    public class ApiResponse<T>
+    {
+        public HttpStatusCode Code { get; }
+        public string Body { get; } = string.Empty;
+        public int Remaining { get; }
+        public int Reset { get; }
+        public Dictionary<string, string> Headers { get; }
```

Comment:
What is this used for?

Reply:
It wasn't being used, removed in 7327c8a2b6b17795f814c6ea88f356c9ad783a45